jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/controller/TopicControllerTest.java

```diff
@@ -200,13 +202,22 @@ public void createTopicShouldNotPassAndMustShowTopicErrorIfItIsInvalid() throws
         BindingResult result = mock(BindingResult.class);
         when(result.hasErrors()).thenReturn(true);
         when(branchService.get(BRANCH_ID)).thenReturn(branch);
-        when(breadcrumbBuilder.getForumBreadcrumb(branch)).thenReturn(new ArrayList<Breadcrumb>());
+        List<Breadcrumb> breadcrumbs = new BreadcrumbBuilder().getNewTopicBreadcrumb(branch);
+        when(breadcrumbBuilder.getNewTopicBreadcrumb(branch)).thenReturn(breadcrumbs);
 
         ModelAndView mav = controller.createTopic(getDto(), result, BRANCH_ID);
 
         verify(branchService).get(BRANCH_ID);
         verify(breadcrumbBuilder).getNewTopicBreadcrumb(branch);
-        //
+
+        List<Breadcrumb> breadcrumbsFromModel = (List) mav.getModel().get("breadcrumbList");
+        for (int i = 0; i < breadcrumbs.size(); i++) {
+            assertEquals(breadcrumbs.get(i).getValue(), breadcrumbsFromModel.get(i).getValue());
+            if(i == 1)
```

Comment:
I have nothing against copy-paste here. I just want to know why we need theses `if` blocks and asserts inside loop